### PR TITLE
Add GSK icon

### LINKED ISSUE #5301: Request: GSK
**Brand Name:** GSK (or something to be "GlaxoSmithKline")
**Website:** https://www.gsk.com/
**Color:** `#f36633`
**Official resources for icon and color:** https://www.google.com/search?q=GSK&tbm=isch

### PR DESCRIPTION
**Pull request by RPicster**
**Issue:** closes #5301

**Similarweb rank: 39,230**

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Nothing special to say here 🥳

### CODE REVIEW

**Comment by RPicster** [comment]
Hmmm, I can't seem to get this icon to be correctly linted... can someone with more experience give me a hint? 💪

**Review by LitoMore — APPROVED** [review]
CleanShot 2022-10-14 at 01 01 05@2x

@RPicster Looks good.

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4738,6 +4738,16 @@
             "source": "https://github.com/gruntjs/gruntjs.com/blob/70f43898d9ce8e6cc862ad72bf8a7aee5ca199a9/src/media/grunt-logo-no-wordmark.svg",
             "guidelines": "https://github.com/gruntjs/grunt-docs/blob/main/Grunt-Brand-Guide.md"
         },
+        {
+            "title": "GSK",
+            "hex": "F36633",
+            "source": "https://www.gskbrandhub.com/",
```
Comment (LitoMore) [inline _data/simple-icons.json:4744]:
```suggestion
            "source": "https://www.gskbrandhub.com",
```

**Comment by RPicster** [comment]
Great! Just like in the other PRs, I'd ask for the `hacktober-accepted` label so it counts towards my profile. @adamrusted labeled the issue with `hacktober`

**Pull request merged**